app/controllers/admin/postcodes_controller.rb

```diff
@@ -39,8 +39,12 @@ def postcodes
     def flash_message
```

Comment:
This method looked a bit more generic and safe to use before, but now it is tightly coupled with the `update_postcode_courts` method and will explode if it gets called before it or elsewhere. So, maybe rename the method to be more clear on where it should be used?

Also we could have methods like:
```
def total_updated_postcodes
  (@updated_postcodes || []).count
end
```

and use them instead of the `@updated_postcodes.count}` in the line 45?

Also aren't the lines 42 and 45 the same? Can we dry them out?

Finally, do you think we could use the `add_flash_types` [method](http://api.rubyonrails.org/v4.2.9/classes/ActionController/Flash/ClassMethods.html) for a more elegant solution?